Base/QTCore/qSlicerUtils.h

```diff
@@ -53,6 +53,8 @@ class Q_SLICER_BASE_QTCORE_EXPORT qSlicerUtils
   /// regular expression: "(lib)?qSlicer.+Module\\.(so, dll, dylib)"
   static bool isLoadableModule(const QString& filePath);
 
+  static bool isTestingModule(const QStringList& categories);
```

Comment:
`hasTestingCategory` ?

Reply:
This method determines if the module is for testing only, the current implementation is to check the category but in the future it may use different mechanisms.

I'll add a comment to clarify this.

Reply:
Thanks.

Would it make sense to have the following signature then:

```
static bool isTestingModule(qSlicerAbstractCoreModule* module);
```